libs/sdk-ui-pivot/src/PivotTable.tsx

```diff
@@ -46,14 +46,15 @@ import { AVAILABLE_TOTALS } from "./impl/base/constants";
  * @returns new prepared execution
  */
 function prepareExecution(props: IPivotTableProps): IPreparedExecution {
-    const { backend, workspace, filters, sortBy = [] } = props;
+    const { backend, workspace, filters, sortBy = [], execConfig = {} } = props;
```

Comment:
The dafault to {} can cause problems, please remove it (similarly to the other charts).

Reply:
unfortunately linting complains about it being undefined if you want it to be removed.